samples: wifi: nrf_cloud: turn multi-service sample into WiFi nRF Cloud sample (#28546)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Reviewer: ```suggestion
* Removed:
   
  * All nRF Cloud REST samples as the nRF Cloud REST library has been removed.
  * The Cellular: nRF Cloud multi-service sample.
```
Could move the removed entry here.

* :ref:`fast_pair_locator_tag` sample:

  * Updated the UI thread handling for reference board targets with a speaker by moving the speaker control into the common indication thread.
    The signaling is now done using the :ref:`zephyr:events` API.

  * Fixed the ringing status indications with the green LED flashes for reference board targets.
    A ringing status indication was often skipped during the motion detection event.

Cellular samples
----------------

* Removed:

  * The :ref:`nrf_cloud_multi_service` sample (formerly ``samples/cellular/nrf_cloud_multi_service``).
    The sample has been rebranded and moved to :file:`samples/wifi/nrf_cloud`.

>     See :ref:`wifi_nrf_cloud` for the new location.

Reviewer 2: ```suggestion
* Removed the Cellular: nRF Cloud multi-service sample.
  The sample has been rebranded and moved to :file:`samples/wifi/nrf_cloud`.
  See :ref:`wifi_nrf_cloud` for the new sample.
```
You need to open the name of the removed sample doc, instead of linking to it, as the link would not work any longer.

  * Updated to use the :ref:`lib_nrf_cloud_coap` library instead of the removed nRF Cloud REST library.

* Removed the Cellular: nRF Cloud multi-service sample.

> 

Reviewer: ```suggestion
```

Wi-Fi samples
-------------

* Added:

  * The :ref:`wifi_nrf_cloud` sample (``samples/wifi/nrf_cloud``), rebranded from the former :ref:`nrf_cloud_multi_service` sample.

>     This sample targets nRF7x hardware using Wi-Fi as the transport instead of LTE.

Reviewer 2: ```suggestion
* Added the :ref:`wifi_nrf_cloud` sample, rebranded from the former Cellular: nRF Cloud multi-service sample.
  This sample targets nRF7x hardware using Wi-Fi as the transport instead of LTE.
```

Wi-Fi samples
-------------

> * Added the :ref:`wifi_nrf_cloud` sample, rebranded from the former Cellular: nRF Cloud multi-service sample.

Reviewer 3: @peknis do you think we should specify in the cellular samples that the Multi-Service sample was removed/converted or is this enough?

Reviewer 2: I agree. We could simply say that the multiservice sample was removed. And have an entry for adding the Wi-Fi Cloud sample.

Author: There is this in changelog. Not good enough?
```
* Removed the Cellular: nRF Cloud multi-service sample.
  The sample has been rebranded and moved to :file:`samples/wifi/nrf_cloud`.
  See :ref:`wifi_nrf_cloud` for the new sample.
```

Reviewer 2: We do not need to say anything about the rebranding and moving. The multiservice sample is removed and that's it. The new sample is mentioned under the Wi-Fi samples, right?

Author: OK, done!

Author: (the new is mentioned under Wi-Fi samples yes indeed)

Wi-Fi samples
-------------

* Added the :ref:`wifi_nrf_cloud` sample, rebranded from the former Cellular: nRF Cloud multi-service sample.

>   This sample targets nRF7x hardware using Wi-Fi as the transport instead of LTE.

Reviewer 4: I'd suggest to use nRF7002 directly as the only one currently capable of establishing connection to cloud:
```suggestion
  This sample targets nRF7002 Wi-Fi companion IC using Wi-Fi as the transport instead of LTE.
```

Wi-Fi samples
-------------

> * Added the :ref:`wifi_nrf_cloud` sample, rebranded from the former Cellular: nRF Cloud multi-service sample.

Reviewer 5: May be "Refactored" instead of "rebranded"?

Author: done

>   This sample targets nRF7002 Wi-Fi companion IC using Wi-Fi as the transport instead of LTE.

Reviewer 5: ```suggestion
  This sample targets the nRF7002 Wi-Fi companion IC, using Wi-Fi as the transport instead of LTE.
```
